File: Java/Technology Fundamentals with Java/Exercises Java Code/src/MethodExercise/PalindromeIntegers.java
package MethodExercise;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class PalindromeIntegers {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        String input = "";
        while (!"END".equals(input = reader.readLine())) {

            printIsPalindrom(input);
        }
    }

    private static void printIsPalindrom(String input) {
        boolean isPalindrom = true;
        for (int i = 0; i <= input.length()/2; i++) {

            char leftSymbol = input.charAt(i);
            char rigthSymbol = input.charAt(input.length() - i - 1);
            if (leftSymbol != rigthSymbol) {
                isPalindrom = false;
            }
        }
        System.out.println(isPalindrom);
    }
}

File: C#/Technology Fundamentals with C#/04. Methods - Functions - Exercise/09. Palindrome Integers/PalindromeIntegers.cs
using System;

public class PalindromeIntegers
{
    public static void Main()
    {
        while (true)
        {
            string word = Console.ReadLine();
            if (word == "END")
            {
                break;
            }
            string reversedWord = ReversedWord(word);

            bool isPalindrom = IsPalindrom(word, reversedWord);
            Console.WriteLine(isPalindrom ? "true" : "false");
        }
    }

    private static string ReversedWord(string word)
    {
        string reversedWord = string.Empty;
        for (int i = word.Length - 1; i >= 0; i--)
        {
            reversedWord += word[i];
        }
        return reversedWord;
    }

    private static bool IsPalindrom(string word, string reversedWord)
    {
        return word == reversedWord ? true : false;
    }
}
